lms/templates/instructor/instructor_dashboard_2/metrics.html

```diff
@@ -146,6 +146,30 @@
         metrics_overlay.find('.metrics-student-grades').show();
         metrics_overlay.show();
       });
+      
+      // Click handler for x axis major ticks
+      $('.metrics-container').on("click", '.stacked-bar-graph-axis .tick.major', function () {
+
+        var stackedBarElement;
+        var xValue;
+        var moduleId;
+        var tickIdentifier = d3.select(this).text();
+
+        var stackedBars = $(this).parent().siblings();
+        for (var sbIndex in stackedBars) {
+          var stackedBarElement = stackedBars[sbIndex];
+          if (stackedBarElement.getAttribute("class") == 'stacked-bar') {
+            xValue = d3.select(stackedBarElement).data()[0].xValue;
+            if (xValue == tickIdentifier) {
+              moduleId = d3.select(stackedBarElement).data()[0].stackData[0].module_url;
+              break;
+            }
+          }
+        }
+
+        var url = '//' + "${section_data['lms_base']}" + '/courses/' + "${section_data['course_id']}" + '/jump_to/' + moduleId;
```

Comment:
Instead of writing this as a single, long line, I find this type of multistring concatenation to be easier to read/maintain (and possible more performant) by using the `Array.join` idiom:

```javascript
url = [
    '/',
    '${section_data['lms_base']}',
    'courses',
    '${section_data['course_id']}',
    'jump_to',
    moduleId
].join('/');
```

It's also less prone to errors when someone mistakenly omits one of the `/`s.